tests/Support/SupportCollectionTest.php

```diff
@@ -395,23 +395,6 @@ public function testChunk()
         $this->assertEquals([10], $data[3]->toArray());
     }
 
-    public function testEvery()
```

Comment:
Lots of code depends on `Collection::every`. Even if you duplicate the test for `Array:every`, it should be left to prevent future unexpected behaviour in collections.